src/apps/relay/mainrelay.c

```diff
@@ -860,18 +867,20 @@ int get_a_local_relay(int family, ioa_addr *relay_addr) {
 }
 
 //////////////////////////////////////////////////
-
+// clang-format off
+#define _STRVAL(s) _STR(s)
+#define _STR(s) #s
 static char Usage[] =
     "Usage: turnserver [options]\n"
     "Options:\n"
     " -d, --listening-device	<device-name>		Listener interface device (NOT RECOMMENDED. Optional, Linux "
     "only).\n"
-    " -p, --listening-port		<port>		TURN listener port (Default: 3478).\n"
+    " -p, --listening-port		<port>		TURN listener port. Default: " _STRVAL(DEFAULT_STUN_PORT) ".\n"
```

Comment:
The changes to stamping the port number into the help text would be lovely to have as a separate PR

Same with disabling prometheous at compile time, though that's a much bigger mess.

Reply:
Hmmm, maybe. But hardcoding the port stuff and then rewrite it again is takes too much time w/o any benefit. Not sure, what is meant with "disabling prometheous at compile time".